examples/internal/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -455,6 +481,13 @@ service ABitOfEverythingService {
 			body: "book"
 		};
 	}
+	rpc UpdateBook(UpdateBookRequest) returns (Book) {
+		option (google.api.http) = {
+		  patch: "/v1/{book.name=publishers/*/books/*}"
+		  body: "book"
+		};
+		option (google.api.method_signature) = "book,update_mask";
```

Comment:
Ditto new dependency here, I don't understand what this adds, since we're not parsing this annotation anywhere.

Reply:
This was to match the [AIP for Updated](https://google.aip.dev/134#guidance), removing since there is no change in behavior